### PR TITLE
Use laboratory to allow us to control feature flags

### PR DESCRIPTION
**Pull request by arkadyan**
Go to /_flags to enable or disable active flags.

Asana ticket: [Feature flags](https://app.asana.com/0/1112935048846093/1132105244340199)

### CODE REVIEW

**Comment by katehedgpeth** [comment]
do you want to try connecting the front end as part of this PR, or are you planning to do that separately?

**Comment by arkadyan** [comment]
@katehedgpeth That's a good point. I should just do that now.

**Comment by arkadyan** [comment]
@katehedgpeth I added control over the headway colors using the flag. Codecov seems like it's having trouble, but could you take another look when you get a chance, please?

File: lib/skate_web/templates/page/index.html.eex [hunk 1]
```diff
@@ -1 +1 @@
-<div id="app" data-user-token="<%= assigns[:user_token] %>"></div>
+<div id="app" data-user-token="<%= assigns[:user_token] %>" data-laboratory-features="<%= Jason.encode!(assigns[:laboratory_features]) %>"></div>
```
Comment (katehedgpeth) [inline lib/skate_web/templates/page/index.html.eex]: `:laboratory_features` will always be assigned, right? in which case it's safe to use `@laboratory_features` in this template

**Comment by arkadyan** [comment]
@katehedgpeth good catch, thanks, I added that check.

**Pull request merged**